frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -135,6 +135,17 @@ export const trendsLogic = kea<trendsLogicType>({
                 }
             },
         ],
+        aggregationGroupTypeIndex: [
```

Comment:
Note that this selector leads to different behavior to what we discussed.

I suggest:
- If N group lines: `Click to view GroupType`
- If N person lines: `Click to view people`
- If mixed: `Click to view`

Reply:
I'm probably missing something here, but it looks like this check accounts for this case. 

If every event or action is the same as the first one's `math_group_type_index`, we show an aggregation label. If the first `math_group_type_index` is undefined, all lines are person lines. If the first `math_group_type_index` is a number, all lines are group lines.

If any event or action doesn't satisfy the above condition, it is mixed and we return -1. 

Is there an edge case I'm missing?